canonical-web-and-design/dqlite.io
# Updates docker yarn run to use --production

## Done

- Updates docker yarn run to use --production

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check nothing is broken
